class/defaults.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
   openshift4_nodes:
-    namespace: openshift-machine-api
+    machineApiNamespace: openshift-machine-api
```

Comment:
Isn't that a breaking change?

Reply:
It shouldn't be, since changing the namespace would break the config anyway.

Reply:
I meant like if somewhere in the hierarchy the `namespace` param has been set with a custom value, it would be ineffective and revert to default value if the parameter got renamed?

Reply:
sure, but if somebody set the `namespace` parameter with a custom value, they've got a non-functional config anyway.